.gitignore

```diff
@@ -9,7 +9,8 @@ osrm/
 *.zip
 npm-debug.log
 osrm-backend/*
-*graphhopper*
+*graphhopper/*
+*graphhopper.*
```

Comment:
what does this catch that wasn't previously caught by `*graphhopper*`?

Reply:
This is actually because it was ignoring the new GraphhopperUtils.js file